Makefile

```diff
@@ -251,17 +251,23 @@ check-translate:
 	find $(TRANSLATE_SOURCES) -type d \( $(TRANSLATE_SOURCES_EXC) \) -prune -o -type f \( -iname "*.c" -o -iname "*.h" \) -print | (LC_ALL=C sort) | xgettext -f- -L C -s --add-location=file --keyword=translate --keyword=MP_ERROR_TEXT -o circuitpython.pot.tmp -p locale
 	$(PYTHON) tools/check_translations.py locale/circuitpython.pot.tmp locale/circuitpython.pot; status=$$?; rm -f locale/circuitpython.pot.tmp; exit $$status
 
+.PHONY: stubs
 stubs:
-	@mkdir -p circuitpython-stubs
+	@rm -rf circuitpython-stubs
+	@mkdir circuitpython-stubs
 	@$(PYTHON) tools/extract_pyi.py shared-bindings/ $(STUBDIR)
 	@$(PYTHON) tools/extract_pyi.py extmod/ulab/code/ $(STUBDIR)/ulab
 	@$(PYTHON) tools/extract_pyi.py ports/atmel-samd/bindings $(STUBDIR)
 	@$(PYTHON) tools/extract_pyi.py ports/raspberrypi/bindings $(STUBDIR)
-	@$(PYTHON) setup.py -q sdist
+	@cp setup.py-stubs circuitpython-stubs/setup.py
+	@cp README.rst-stubs circuitpython-stubs/README.rst
```

Comment:
Why not copy the README.rst itself? There is a lot of duplication now.

Reply:
My thinking was that we wanted the stubs package to clearly identify itself, which the "main" README.rst would not.  I can copy it if that makes more sense.

Reply:
Ya, I'm fine with that. I think we'll want to strip it down more aggressively though and then link to the main one instead of duplicating a bunch of info.